mrblib/haconiwa/runner.rb

```diff
@@ -488,17 +488,23 @@ def persist_namespace(pid, namespace)
       end
     end
 
+    def process_exists?(pid)
+      begin
+        ::Process.kill(0, pid)
+      rescue
+        false
+      end
+    end
+
     def confirm_existence_pid_file(pid_file)
-      if File.exist? pid_file
-        if ::Process.kill(0, File.read(pid_file).to_i)
-          raise "PID file #{pid_file} exists. You may be creating the container with existing name #{@base.name}!"
-        else
-          begin
-            File.unlink(pid_file)
-            Haconiwa::Logger.debug("Since the process does not exist, delete the PID file #{pid_file}")
-          rescue
-            raise "Failed to delete PID file #{pid_file}."
-          end
+      if File.exist? pid_file && process_exists?(File.read(pid_file).to_i)
```

Comment:
This may be parsed `File.exist?(pid_file && process_exists?(File.read(pid_file).to_i))`, so test is failing